northstar_tests/src/runtime.rs

```diff
@@ -212,23 +212,19 @@ impl Northstar {
 
     // TOOD: Queue the notifications in the runtime struct. Currently there's a race
     // if the notification is faster.
-    pub async fn assume_notification<F>(&mut self, mut pred: F, timeout: u64) -> Result<()>
+    pub async fn assume_notification<F>(&mut self, mut pred: F, timeout: Duration) -> Result<()>
```

Comment:
I'd not use `Duration` here since it's bulky to type and also keep it in sync with the logger assumption API that accepts seconds also.

Reply:
changed back but i think such assumptions and internal conversions are not nice for a API